### PR TITLE
Test scripts for CI

### PR DESCRIPTION
**Pull request by gunt0001**
[Tracking issue](https://devtopia.esri.com/runtime/kotlin/issues/4014)

### Description:

Sets up scripts to build & run tests on the Jenkins CI environment. This follows closely the pattern applied to runing Kotlin SDK tests on Jenkins, i.e. have a `ci/test` folder with scripts for set up, build, test and vTest.

### Changes:
- Introduced new `test` folder under `ci`
- added `common_test.sh`, `setup.sh`, `build.sh`, `test.sh` and `vtest.sh`
- added readme

### CODE REVIEW

**Comment by gunt0001** [comment]
@sorenoid - please review

File: ci/test/build.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+source $(dirname ${BASH_SOURCE})/../common.sh
+
+cd ${apps_path}/arcgis-maps-sdk-kotlin-toolkit
+echo "$(pwd)"
```
Comment (chri7325) [inline ci/test/build.sh:6]: Should we remove these debugging echos?

File: ci/test/common_test.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+function _reboot_device() {
+  # Check for adb executable
+  if [ ! -f "${adb}" ]; then
+    echo "error: Couldn't find the adb executable. Please verify android sdk tools installation"
+    exit 1
+  fi
+  _log "Restarting test device"
+  ${adb} reboot
+
+  # wait-for-any-device typically doesn't wait long eough for the device to be truly
```
Comment (chri7325) [inline ci/test/common_test.sh:24]: enough

**Review by chri7325 — COMMENTED** [review]
@gunt0001 since toolkit tests are going to run on the kotlin test server, I'm wondering if we should just get the steps to build and run them in the internal Kotlin repo. Duplicating all scripts seems like a maintenance nightmare where instead we can just clone the toolkit and look at augmenting the Kotlin scripts to call the toolkit ones instead.

**Comment by gunt0001** [comment]
We will manage the CI scripts internally, closing...

**Pull request closed without merging**